.github/workflows/cypress-workflow.yml

```diff
@@ -7,9 +7,9 @@ on:
     branches:
       - "*"
 env:
-  OPENSEARCH_DASHBOARDS_VERSION: '2.5.0'
-  OPENSEARCH_VERSION: '2.5.0-SNAPSHOT'
-  SECURITY_ANALYTICS_BRANCH: '2.5'
+  OPENSEARCH_DASHBOARDS_VERSION: '2.6.0'
```

Comment:
You'll want to change this from `2.6.0` to `2.6`. 
`2.6.0` will cause the action to look for a `tag` with that name to use as its `ref`, whereas `2.6` will cause it to look for a `branch` with that name. The `2.6.0` tag isn't available yet as we haven't completed the release.

Reply:
aah, thanks!